### PR TITLE
Bump Serving to v1.5

### PR DESCRIPTION
**Pull request by skonto**
## Proposed Changes

- The usual as in https://github.com/openshift-knative/serverless-operator/pull/1647

### CODE REVIEW

**Comment by skonto** [comment]
```
{"level":"error","ts":"2022-09-25T21:56:55.098Z","logger":"knative-operator","caller":"knativeserving/reconciler.go:294","msg":"Returned an error","knative.dev/pod":"knative-operator-79657bc6b5-nr5kz","knative.dev/controller":"knative.dev.operator.pkg.reconciler.knativeserving.Reconciler","knative.dev/kind":"operator.knative.dev.KnativeServing","knative.dev/traceid":"9afd61b1-3007-47d7-a66d-7e3e00f05078","knative.dev/key":"knative-serving/knative-serving","targetMethod":"ReconcileKind","error":"failed to apply non rbac manifest: admission webhook \"config.webhook.serving.knative.dev\" denied the request: validation failed: the update modifies a key in \"_example\" which is probably not what you want. Instead, copy the respective setting to the top-level of the ConfigMap, directly below \"data\"","stacktrace":"knative.dev/operator/pkg/client/injection/reconciler/operator/v1alpha1/knativeserving.
```

**Comment by skonto** [comment]
ci/prow/4.11-operator-e2e-aws-ocp-411: 
```
10:03:10.140 DEBUG:   [[ $(oc get pods -n knative-serving --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]] : Waiting until non-zero (max 600 sec.)
.........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
10:13:54.998 ERROR:   Time out of 600 exceeded
```

**Comment by skonto** [comment]
Run the operator tests locally against a 4.11 cluster:
```
DEBUG   14:56:44.048 Debug log (set -x) is written to: /tmp/tmp.OGsaOTCnZS/debuglog-teardown.sh.log
DEBUG   14:56:44.050 Registering error handler: stacktrace
WARNING 14:56:44.052 😭  Teardown Serverless...
INFO    14:56:46.618 Removing KnativeServing CR
knativeserving.operator.knative.dev "knative-serving" deleted
INFO    14:57:00.130 Ensure no knative serving pods running
DEBUG   14:57:00.138 [[ $(oc get pods -n knative-serving --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]] : Waiting until non-zero (max 600 sec.) ...................... done
namespace "knative-serving" deleted
IN
```

File: hack/lib/serverless.bash [hunk 1]
```diff
@@ -327,7 +327,7 @@ function teardown_serverless {
     oc delete knativeserving.operator.knative.dev knative-serving -n "${SERVING_NAMESPACE}"
   fi
   logger.info 'Ensure no knative serving pods running'
-  timeout 600 "[[ \$(oc get pods -n ${SERVING_NAMESPACE} --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]]"
+  timeout 900 "[[ \$(oc get pods -n ${SERVING_NAMESPACE} --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]]"
```
Comment (skonto) [inline hack/lib/serverless.bash:330]: Testing to see if this is ever deleted.

**Comment by skonto** [comment]
Ok found the issue with upgrade tests we need quay promotion because during upgrades we test with that instead:
```
[migrate]","reason":"ContainersNotReady","status":"False","type":"ContainersReady"},{"lastProbeTime":null,"lastTransitionTime":"2022-09-26T15:39:18Z","status":"True","type":"PodScheduled"}],"containerStatuses":[{"image":"quay.io/openshift-knative/knative-serving-storage-version-migration:v1.5.0","imageID":"","lastState":{},"name":"migrate","ready":false,"restartCount":0,"started":false,"state":{"waiting":{"message":"Back-off pulling image \"quay.io/openshift-knative/knative-serving-storage-version-migration:v1.5.0\"","reason":"ImagePullBackOff"}}}],"hostIP":"10.0.128.38","phase":"Pending","podIP":"10.129.2.16","podIPs":[{"ip":"10.129.2.16"}],"qosClass":"Burstable","startTime":"2022-09-26T15:39:18Z"}}] != \[\] ]]
+ seconds=899
```

**Comment by skonto** [comment]
infra
> Total download size: 95 M
Downloading packages:
No Presto metadata available for updates
https://rpm.nodesource.com/pub_14.x/el/7/x86_64/nodejs-14.18.0-1nodesource.x86_64.rpm: [Errno 14] curl#18 - "transfer closed with 2247078 bytes remaining to read"
Trying other mirror.
Error downloading packages:
  2:nodejs-14.18.0-1nodesource.x86_64: [Errno 256] No more mirrors to try.

**Comment by skonto** [comment]
@pierDipi is `4.11-upstream-e2e-kafka-aws-ocp-411` wip?

**Comment by pierDipi** [comment]
> @pierDipi is `4.11-upstream-e2e-kafka-aws-ocp-411` wip?

yes, please ignore

**Comment by skonto** [comment]
@pierDipi or @mgencur want to stamp this one? :pray:

**Pull request merged**